novu-style.gemspec

```diff
@@ -27,7 +27,7 @@ Gem::Specification.new do |spec|
   spec.executables   = spec.files.grep(%r{^exe/}) { |f| File.basename(f) }
   spec.require_paths = ['lib']
 
-  spec.add_dependency 'rubocop', '~> 0.51.0'
+  spec.add_dependency 'rubocop', '~> 0.52.1'
```

Comment:
Do we even want to allow any difference in rubocop version with `~>`?

Reply:
It was done this way to exactly match codeclimate rubocop configuration.
https://github.com/codeclimate/codeclimate-rubocop/blob/master/Gemfile#L9